sympy/core/relational.py

```diff
@@ -1075,9 +1079,281 @@ def _n2(a, b):
             return dif
 
 
+# helpers for _eval_is_ge below
+_invpow = lambda p: Pow(p.base, -p.exp, evaluate=False)
+
+_small_pos = lambda x: fuzzy_and((
+    is_ge(x, S.Zero), is_ge(S.One, x)))
+
+_small_neg = lambda x: fuzzy_and((
+    is_ge(x, S.NegativeOne), is_ge(S.Zero, x)))
+
+_big_ppow = lambda b, e: fuzzy_or((
+        fuzzy_and((b.is_zero, e.is_zero)),  # 1
+        fuzzy_and((Mod(e, 2) == 0, fuzzy_or((
+        is_ge(S.NegativeOne, b) and is_ge(e, S.Zero),
+        _small_neg(b) and is_ge(S.Zero, e))))),
+    _small_pos(b) and is_ge(S.Zero, e),
+    is_ge(b, S.One) and is_ge(e, S.Zero)))
+
```

Comment:
This kind of thing is definitely better as a proper function with a docstring rather than a lambda function. If the function is internal then it does not need to have a "full" docstring but at least an explanation of what it tests. I don't know what `_big_ppow` means and it would take a long time to parse it from the code here!

Reply:
I'm in the mid-game right now. I really hope this can be a lot simpler.

Reply:
for the record, the helpers identify whether a power is positive or negative and big (>= 1) or small (between 0 and 1).